.github/workflows/main_housingrepairsschedulingapi.yml

```diff
@@ -5,9 +5,12 @@ name: Build and deploy ASP.Net Core app to Azure Web App - HousingRepairsSchedul
 
 on:
   push:
+    branches: [ main ]
+  pull_request:

```

Comment:
Is this temporary? i.e. just to test

Keeping it would mean all PRs are pushed to _production/live_, which is not desirable as it could disrupt the live service.

Reply:
currently the prs arent being deployed with this ci?

Reply:
The build step only happens when you deploy to main and deployment is dependent on the build step

Reply:
Thanks for the clarification - my misunderstanding.